### DISCUSSION TITLE
How long can a BTC transfer stay unconfirmed?

### DISCUSSION THREAD

**Discussion by iggy097** [btcpayserver/btcpayserver#5839] [General]
Sent my BTC from my BTCPayserver to my cashapp wallet but used a low fee and now it's just sitting unconfirmed for days. Any ideas how long before it gets confirmed or returned?
[https://mempool.space/tx/73747e9005cb648c4aa58b25de707ad6aa913afa78490de7d17d58e2ee161e9e](url)

**Comment by fedex1** [comment]
According to what I read it can stay unconfirmed forever. 

I looked at that transaction and it appears that it is transferring about $380 USD and there's some option on there to use an accelerator and they want to charge about $50 USD to accelerate it.  A 13% charge. 

That looks like quite a business. 😁

**Comment by iggy097** [comment]
No wonder this will never have widespread adoption. My BTC is just stuck in limbo now for however long because I had no idea that that was even possible. Perfect.

**Reply by dennisreimann** [reply]
The transaction has RBF enabled, so you can increase the fee and acelerate it yourself that way. Unfortunately we do not support it in BTCPay Server yet. But you could import the wallet e.g. into Sparrow and do it that way. We plan to support RBF with v1.14

Depending on what capabilities CashApp offers, you could also use CPFP to do another transaction from the CashApp wallet to , spending the unconfirmed input to yourself, but with a higher fee. This way the low fee transaction would piggy-back on the higher fee transaction (CPFP = Child pays for parent).

**Reply by iggy097** [reply]
Sparrow seems way too complicated for me. I do have Ledger - is it possible to this with Ledger Live?
